backend/nlp/handler.py

```diff
@@ -109,16 +109,20 @@ def analyze_reviews(reviews_list):
                 if sentiment_scores['neg'] > bad:
                     bad = sentiment_scores['neg']
 
-                #if abs(sentiment_scores['compound']) > abs(compound):
+                neutral += sentiment_scores['neu']
                 compound += sentiment_scores['compound']
 
+            compound /= len(sentences)
+            neutral /= len(sentences)
+
+            review['neuSentiment'] = int(neutral * 100)
             review['negSentiment'] = int(bad * 100)
             review['posSentiment'] = int(good * 100)
             review['compSentiment'] = int(compound * 100)
 
 
             if review["review"]["title"].strip() == "":
-                review["review"]["title"] = "No Title Given"
```

Comment:
Let's just remove the "title" attribute if we don't have any data to put there - DynamoDB is fine with that, since title isn't a key.